src/lib/dimension.js

```diff
@@ -62,3 +62,35 @@ export function calculateWidth(calMethod = 'scroll') {
   }
   return getWidth[calMethod]();
 }
+
+/**
+ * This function returns the offset height of the given node relative to the top of document.body
+ */
+export function getOffsetToBody(node, offset = 0) {
```

Comment:
does this need to e exported?

Reply:
not necessary, but may be good to make testing easier.